src/ObjLoading/Parsing/LocalizeFile/LocalizeFileParserState.h

```diff
@@ -2,15 +2,15 @@
 #include "Game/GameLanguage.h"
 #include "Localize/LocalizeFile.h"
 
+#include <map>
 #include <unordered_set>
-#include <vector>
 
 class LocalizeFileParserState
 {
 public:
     bool m_end;
 
-    std::vector<LocalizeFileEntry> m_entries;
+    std::map<std::string, std::string> m_entries;
```

Comment:
After you removed the `LocalizeFileEntry` struct here i think you removed all usages of it. You can probably remove it then.